imageio/imageio-psd/src/main/java/com/twelvemonkeys/imageio/plugins/psd/PSDLayerInfo.java

```diff
@@ -191,4 +232,22 @@ public String toString() {
         builder.append("]");
         return builder.toString();
     }
+
+    enum SectionDividerSetting {
+        LAYER(0), OPEN_FOLDER(1), CLOSED_FOLDER(2), BOUNDING_SECTION_DIVIDER(3);
+        SectionDividerSetting(int value) { this.value = value;}
+
+        private final int value;
+        public int value() { return value; }
+
+        public static SectionDividerSetting fromInt(int value){
+            for(SectionDividerSetting rt : SectionDividerSetting.values()){
+                if(rt.value == value){
+                    return rt;
+                }
+            }
+
+            return null;
+        }
+    }
```

Comment:
This block has code style issues.

In addition to the above, use one enum constant per line.